test/support/test_helpers.ex

```diff
@@ -28,7 +33,8 @@ defmodule RemoteRetro.TestHelpers do
       User.changeset(%User{}, user_params)
       |> Repo.insert!
 
-    Map.put(context, :user, user)
```

Comment:
hey @tnewell5. no need to add an id to the user, as `user` will already have one coming back from the database.